site/content/contribute/getting-started/labels.md

```diff
@@ -0,0 +1,58 @@
+---
+title: "Labels"
+date: 2018-03-06T00:00:00-04:00
+weight: 5
+subsection: Getting Started
+---
+
+We leverage [GitHub labels](https://help.github.com/en/articles/about-labels) to track the details and lifecycle of issues and pull requests.
+
+# Issue Labels
+* `Area/<name>`: Involves changes to the named area (APIv4, Add E2E Tests, Plugins, etc.)
+* `Difficulty/1:easy`: Easy ticket.
+* `Difficulty/2:medium`: Medium ticket.
+* `Difficulty/3:hard`: Hard ticket.
+* `First Good Issue`: Suitable for first-time contributors.
+* `Framework/<name>`: Requires using the named framework (ReactJS, Redux, etc.)
```

Comment:
In the dev meeting where the new labelling system was introduced, the concern was brought up that Redux and others are not a framework.

My proposal is to use the term `Tech/<name>` instead of `Area` and `Language`, as it's a generic term and grouping these "things" under a common term is what we want. It would be:
- `Tech/Go`
- `Tech/Redux`
- `Tech/React`
- `Tech/Cypress`
- ...

Thoughts on this? 

Also see https://community.mattermost.com/core/pl/8gp6hojrzpg6ubhiohkhn74r8y

Reply:
`Tech` makes sense as an alternative to `Framework`, but I'm not sure about `Area`. I read the linked thread, but it wasn't clear that `Tech` was meant to replace `Area` too. @hanzei & @jasonblais, can you clarify?

Reply:
It is intended to replace `Framework` and `Language`. @lieut-data